pages/content/amp-dev/documentation/components/reference/amp-date-picker-v0.1@zh_CN.md

```diff
@@ -0,0 +1,726 @@
+
+### 选择类型
+
+`amp-date-picker` 将提供两种类型的日期选择：
+
+- `single`: 在日期选择器中选择一个日期。
+- `range`: 在日期选择器中选择一个日期范围。
+
+#### `type="single"`
+
+通过指定 `type="single"`，日期选择器将附加到单个输入，用户可以选择一个日期。这是默认选择类型。
+
+[sourcecode:html]
+<amp-date-picker type="single" layout="fixed-height" height="360">
+</amp-date-picker>
+[/sourcecode]
+
+<amp-img alt="static single date picker" layout="fixed" src="https://github.com/ampproject/amphtml/raw/master/extensions/amp-date-picker/img/amp-date-picker-single-static.png" width="336" height="370">
+  <noscript>
+    <img alt="static single date picker" src="https://github.com/ampproject/amphtml/raw/master/extensions/amp-date-picker/img/amp-date-picker-single-static.png" width="336" height="370">
+  </noscript>
+</amp-img>
+
+#### `type="range"`
+
+通过指定 `type="range"`，日期选择器将附加到两个输入，并且用户可以选择带有开始日期和结束日期的日期范围。
+
+[sourcecode:html]
+<amp-date-picker type="range" layout="fixed-height" height="360">
+</amp-date-picker>
+[/sourcecode]
+
+<amp-img alt="static single date picker" layout="fixed" src="https://github.com/ampproject/amphtml/raw/master/extensions/amp-date-picker/img/amp-date-picker-range-static.png" width="332" height="373">
+  <noscript>
+    <img alt="static single date picker" src="https://github.com/ampproject/amphtml/raw/master/extensions/amp-date-picker/img/amp-date-picker-range-static.png"  width="332" height="373">
+  </noscript>
+</amp-img>
+
+### 日期格式
+
+`amp-date-picker`属性接受ISO 8601和RFC 5545 RRULE格式的日期。
+
+[ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) 将日期格式设置为 `YYYY-MM-DD`
+并且是电子系统之间共享日期的标准。
+例如， ISO 8601 将日期 February 28 2018 格式化为 `2018-02-28`.
+
+[RFC 5545 Recurrence Rules (RRULEs)](https://icalendar.org/iCalendar-RFC-5545/3-3-10-recurrence-rule.html)
+标准化了用于指定重复日期的格式。
+例如， RFC 5545 将 Halloween 设置为 `RRULE:FREQ=YEARLY;BYMONTH=10;BYMONTHDAY=31`。
+也可以使用更复杂的日期，例如美国感恩节假期，即每年的11月的第四个星期四： `RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=+4TH`。该API不便于记忆，但可以在线使用各种 [RRULE 生成器](https://jakubroztocil.github.io/rrule) 。
+
+## 属性
+
+### mode
+
+指定如何呈现日期选择器。允许的值为：
+
+- **`static`** (默认): 日期选择器呈现为交互式日历视图。
+- **`overlay`**: 日期选择器日历视图不会呈现，直到用户交互在'`<amp-date-picker>`中嵌套必需的输入字段。
+
+### type
+
+指定日期选择器的选择类型。允许的值为：
+
+- **`single`** (默认): 用户可以选择一个日期。
+- **`range`**: 用户可以选择一个日期。
+
+### input-selector
+
+用于单个日期选择器输入的查询选择器。如果省略此选项，则日期选择器会自动生成一个隐藏的输入字段，并为其指定名称 `date` 或 `${id}-date` 使用日期选择器的ID。如果这些冲突中的任何一个与表单中的现有元素冲突，都会发出错误。
+
+当`amp-date-picker`加载时，输入元素的值用于显示初始选择的日期。
+
+通过[`src` attribute](#src)指定 `date` 属性来设置动态的初始日期。
+
+[sourcecode:html]
+<amp-date-picker
+  type="single"
+  mode="overlay"
+  layout="container"
+  input-selector="[name=deliverydate]"
+>
+  <input type="text" name="deliverydate" placeholder="Delivery Date" />
+</amp-date-picker>
+[/sourcecode]
+
+### start-input-selector
+
+日期范围选择器的开始日期输入的查询选择器。如果省略此选项，则日期选择器会自动生成一个隐藏的输入字段，并为其指定名称 `start-date` 或 `${id}-start-date` 使用日期选择器的ID。如果这些冲突中的任何一个与表单中的现有元素冲突，都会发出错误。
+
+当 `amp-date-picker` 加载时，输入元素的值用于显示初始选择开始日期。
+
+通过[`src` attribute](#src-optional) 指定 `startDate` 属性来设置
```

Comment:
This link is broken and makes the check failing. Not your fault but already broken in the source document. This should actually simply link to `#src`.

Reply:
done